include/circt/Dialect/SystemC/SystemCOpInterfaces.td

```diff
@@ -0,0 +1,45 @@
+//===- SystemCOpInterfaces.td - Operation Interfaces -------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This describes the SystemC operation interfaces.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CIRCT_DIALECT_SYSTEMC_SYSTEMCOPINTERFACES
+#define CIRCT_DIALECT_SYSTEMC_SYSTEMCOPINTERFACES
+
+include "mlir/IR/OpBase.td"
+
+def SystemCNameDeclOp : OpInterface<"SystemCNameDeclOp"> {
```

Comment:
This is actually defining an `OpInterface`, yes? Is there a convention for what this should be called as opposed to calling it `SystemCNameDeclOp`? This implies to me that it is an op, when it is actually an opinterface.

Reply:
You're right, the default seems to be to have `OpInterface` at the end, so I'll add the 'Interface'.